snomed/com.b2international.snowowl.snomed.fhir/src/com/b2international/snowowl/snomed/fhir/SnomedValueSetApiProvider.java

```diff
@@ -193,7 +198,7 @@ private ValueSet buildSimpleTypeRefsetValueSet(String componentId, CodeSystemVer
 		
 		return getSimpleTypeRefsetSearchRequestBuilder(componentId)
 			.setExpand("members(expand(referencedComponent(expand(pt()))), limit:"+ all +")")
-			.build(codeSystemVersion.getUri())
+			.build(repositoryId, codeSystemVersion.getPath())
```

Comment:
Please use `com.b2international.snowowl.core.codesystem.CodeSystemVersion.getUri()` instead of `repositoryId`.